tests/macsec/test_macsec.py

```diff
@@ -91,7 +91,7 @@ def test_rekey_by_period(self, duthost, ctrl_links, upstream_links, rekey_period
 class TestDataPlane():
     BATCH_COUNT = 10
 
-    def test_server_to_neighbor(self, duthost, ctrl_links, downstream_links, upstream_links, nbr_device_numbers, nbr_ptfadapter):
+    def test_server_to_neighbor(self, duthost, ctrl_links, downstream_links, upstream_links, nbr_device_numbers, nbr_ptfadapter, tbinfo):
```

Comment:
Why do you involve a new fixture `tbinfo` but you didn't use it in the following code?

Reply:
done